Gemfile

```diff
@@ -65,11 +65,19 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem "rspec-rails"
+  gem "factory_bot_rails"
+  gem 'shoulda-matchers', '~> 5.0'
 end
 
 group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "selenium-webdriver"
   gem "webdrivers"
+  gem "rspec-rails"
+  gem "factory_bot_rails"
+  gem 'shoulda-matchers', '~> 5.0'
 end
+gem 'faker'
+gem 'pry'
```

Comment:
Just a little suggestion: we normally try to arrange the gem alphabetically. It's helpful to track when we have a lot of gems in our project. Thank you!

Reply:
sorted. Thank you